gapic/ads-templates/%namespace/%name/%version/%sub/services/%service/_shared_macros.j2

```diff
@@ -38,32 +38,32 @@
 {% endwith %}{# method_settings #}
 {% endmacro %}
 
-{% macro add_google_api_core_version_header_import(service) %}
+{% macro add_google_api_core_version_header_import(service_version) %}
```

Comment:
Could you add a note at the top that this is intended to be a symlink to the other file, and for now is a duplicate (link to the issue you filed)?

Reply:
Fixed in https://github.com/googleapis/gapic-generator-python/pull/1999/commits/a62b3139fcdbc2f411065d2086926e910cd0000e